terraform/layers/base/vpc.tf

```diff
@@ -0,0 +1,29 @@
+// Get names of the available AZs
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
+locals {
+  azs  = data.aws_availability_zones.available.names
+  cidr = "10.0.0.0/16"
```

Comment:
We are probably better capturing this in a variable. Even though the nature of VPCs mean the CIDR is unlikely to change, if we ever wanted to do a full redeploy on a different CIDR a variable would be better to change rather than directly in the code